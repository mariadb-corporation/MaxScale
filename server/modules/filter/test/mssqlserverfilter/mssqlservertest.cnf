[Mssqlserver]
type=filter
module=mssqlserverfilter
match=('string a' + 'string b' + 'string c')
replace=CONCAT('string a', 'string b', 'string c') 
