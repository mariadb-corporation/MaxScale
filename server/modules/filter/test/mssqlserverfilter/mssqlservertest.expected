SELECT concat('This ','is',' concatenation')